doc/chomp_interface/chomp_interface_tutorial.rst

```diff
@@ -1,58 +0,0 @@
-CHOMP Interface
```

Comment:
Note: If you use `git mv` rather than `mv` when renaming these files then git does a better job of tracking your changes vs the original.

Reply:
Also, you need to test with the `build_locally.sh` script before each push. You didn't rename the tutorial in `index.rst` so the chomp tutorial isn't getting built

Reply:
done. I tested the `build_locally.sh` script and renamed the CHOMP tutorial in the `index.rst` file.